scripts/oes-data-migration-scripts/steps_to_migrate_from_v3.10.x_to_v3.11.txt

```diff
@@ -0,0 +1,10 @@
+Steps to migrate from v3.10.x to v3.11
+
+1. Goto location /opsmx/workdir/datamigration in platform container
+2. You will find a file named : migration_v3.10.x_to_v3.11.py
+3. Run the below command to execute the script.
```

Comment:
These steps are not proper. Write the steps like above one

Reply:
worked on comment

Reply:
Worked on comment